pkg/velaql/providers/query/handler.go

```diff
@@ -446,7 +471,7 @@ func generatorFromService(service corev1.Service) []ServiceEndpoint {
 					Port:     port.NodePort,
 				},
 				Ref: corev1.ObjectReference{
-					Kind:            service.Kind,
```

Comment:
What is this change for? Is there any potential problem for `service.Kind`?

Reply:
It is an empty value by default.